mne/tests/common.py

```diff
@@ -46,3 +47,17 @@ def assert_meg_snr(actual, desired, min_tol, med_tol=500., msg=None):
     snr = np.median(snrs)
     assert_true(snr >= med_tol, 'SNR median %0.2f < %0.2f%s'
                 % (snr, med_tol, msg))
+
+
+def assert_dig_allclose(info_py, info_bin):
+    # test dig positions
+    dig_py = info_py['dig']
+    dig_bin = info_bin['dig']
+    assert_equal(len(dig_py), len(dig_bin))
```

Comment:
kinda, so i just explored the files from where they came, the head digitization has 300k points so it was decimated, but there were two separate ways of doing it, so the those points never matched up. I do have the original `*.elp` that was used in the mne-c conversion and I have the elp text file that is the current format. would you like those?